src/main/java/com/flipkart/ranger/healthcheck/HealthChecker.java

```diff
@@ -41,7 +45,7 @@ public void run() {
             try {
                 healthcheckStatus = healthcheck.check();
             } catch (Throwable t) {
-                logger.error("Error running healthcheck. Setting node to unhealthy");
+                logger.error("Error running healthcheck. Setting node to unhealthy", t);
```

Comment:
Any retries that you want to do here? something like node might be recovering from something or there was just a temporary glitch. it is surely unhealthy but given check() had thrown throwable, it could be anything.